core/runtime/src/main/java/io/quarkus/runtime/logging/SyslogConfig.java

```diff
@@ -0,0 +1,100 @@
+package io.quarkus.runtime.logging;
+
+import java.util.Optional;
+import java.util.logging.Level;
+
+import org.jboss.logmanager.handlers.SyslogHandler.Facility;
+import org.jboss.logmanager.handlers.SyslogHandler.Protocol;
+import org.jboss.logmanager.handlers.SyslogHandler.SyslogType;
+
+import io.quarkus.runtime.annotations.ConfigGroup;
+import io.quarkus.runtime.annotations.ConfigItem;
+
+@ConfigGroup
+public class SyslogConfig {
+
+    /**
+     * If syslog logging should be enabled
+     */
+    @ConfigItem
+    boolean enable;
+
+    /**
+     * The address of the syslog server
+     */
+    @ConfigItem(defaultValue = "localhost")
+    String serverAddress;
+
+    /**
+     * The port the syslog server is listening on
+     */
+    @ConfigItem(defaultValue = "514")
+    int port;
```

Comment:
An alternative to having `serverAddress` + `port` is to have one property of type `InetSocketAddress`.  This also prevents premature resolution of host names compared to using `InetAddress`.

Reply:
It'll be something like this ? 
```
    /**
     * The address and port the syslog server is listening on
     */
    @ConfigItem(defaultValue = "localhost:514")
    InetSocketAddress serverAddress;
```

Reply:
Yes, you could.  Using a host name + port is an alternative to having a single property of type `InetSocketAddress`, and works fine as long as you will only ever have one value.  Once you have more than one value though (for example in round-robin reconnect situations, or multiple bind address situations), then having separate host/port properties fails.  This is why projects like Apache HTTPD, OpenSSH, etc. began allowing `host:port` (i.e. socket address) values for this kind of property.

Reply:
So, we have to use a String config `host:port` property  and write a method which will create `InetSocketAddress` instance ? have you any suggestions to make it more efficient ?

Reply:
Not at all.  Just make the configuration property have a type of `InetSocketAddress` with a default value of `"localhost:514"`.

Reply:
Great, i merged the two properties in a single one called: `quarkus.log.syslog.endpoint`. If you have another suggestion, you're welcome.